doc/source/utilities/diffractionobjectsutility.rst

```diff
@@ -0,0 +1,16 @@
+.. _Diffraction Objects Utility:
+
+Diffraction Objects Utility
+===========================
+
+The ``diffpy.utils.scattering_objects.diffraction_objects`` module provides functions
+for managing and analyzing diffraction data, including angle-space conversions
+and interactions between diffraction data.
+
+- ``q_to_tth()``: Converts an array of q values to their corresponding two theta values, based on specified wavelength.
```

Comment:
we don't need this here.  It will be auto-generated in the API docs, and it is shown as an aexample above.